components/server/src/workspace/ec2-workspace-manager.ts

```diff
@@ -0,0 +1,724 @@
+import * as crypto from 'crypto';
```

Comment:
This manager has defined interface functions, may be we can define a generic ec2-workspace-manager and ec2-windows-manager as interface implementor in next phase.